CHANGELOG.md

```diff
@@ -3,6 +3,81 @@
 Full documentation for MIGraphX is available at
 [https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/](https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/).
 
+## MIGraphX 2.9 for ROCm 6.1.0
+
+### Additions
+
+* Added FP8 support
+* Created a dockerfile with MIGraphX+ONNX Runtime EP+Torch
+* Added support for the `Hardmax`, `DynamicQuantizeLinear`, `Qlinearconcat`, `Unique`, `QLinearAveragePool`, `QLinearSigmoid`, `QLinearLeakyRelu`, `QLinearMul`, `IsInf` operators
+* Created web site examples for `Whisper`, `Llama-2`, and `Stable Diffusion 2.1`
+* Created examples of using the ONNX Runtime MIGraphX Execution Provider with the `InceptionV3` and `Resnet50` models
+* Updated operators to support ONNX Opset 19
+* Enable fuse_pointwise and fuse_reduce in the driver
+* Add support for dot-(mul)-softmax-dot offloads to MLIR
+* Added Blas auto-tuning for GEMMs
+* Added dynamic shape support for the multinomial operator
+* Added fp16 to accuracy checker
+* Added initial code for running on Windows OS
+
+### Optimizations
+
+* Improved the output of migraphx-driver command
+* Documentation now shows all environment variables
+* Updates needed for general stride support
+* Enabled Asymmetric Quantization
+* Added ScatterND unsupported reduction modes
+* Rewrote softmax for better performance
+* General improvement to how quantization is performed to support INT8
+* Used problem_cache for gemm tuning
+* Improved performance by always using rocMLIR for quantized convolution
+* Improved group convolutions by using rocMLIR
+* Improved accuracy of fp16 models
+* ScatterElements unsupported reduction
+* Added concat fusions
+* Improved INT8 support to include UINT8
+* Allow reshape ops between dq and quant_op
+* Improve dpp reductions on navi
+* Have the accuracy checker print the whole final buffer
+* Simplify constant of shape and  constant variable input slices
```

Comment:
Can reword this to: Added support for handling dynamic Slice and ConstantOfShape ONNX operators.